modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/client-extension-entries.json

```diff
@@ -9,7 +9,7 @@
 			"[$DOCUMENT_URL:/site-initializer/documents/company/remote-app/osb-site-initializer-customer-portal/main.js$]",
 			"[$DOCUMENT_URL:/site-initializer/documents/company/remote-app/osb-site-initializer-customer-portal/runtime-main.js$]"
 		],
-		"externalReferenceCode": "ERC-001",
+		"externalReferenceCode": "ERC001",
```

Comment:
@nNilton let's follow Raylife's standard (RAY001) and put CP001.
ok?

Reply:
Ok